cmd/podman/pull.go

```diff
@@ -99,7 +99,7 @@ func pullCmd(c *cli.Context) error {
 		DockerCertPath:              c.String("cert-dir"),
 		DockerInsecureSkipTLSVerify: !c.BoolT("tls-verify"),
 	}
-	if !c.IsSet("tls-verify") {
+	if !c.IsSet("tls-verify") || !c.Bool("tls-verify") {
 		forceSecure = false
```

Comment:
Shouldn't this be 

forceSecurte  = c.Bool("tls-verify")

Reply:
That would also work, let me refactor to do that